src/hotspot/share/opto/phasetype.hpp

```diff
@@ -186,13 +190,12 @@ class PhaseNameValidator {
     }
   }
 
-  bool is_valid() const {
-    return _valid;
-  }
-
-  const char* what() const {
-    return _bad;
+  const BitMap& phase_name_set() const {
+    assert(is_valid(), "Use of invalid phase name set");
+    return _phase_name_set;
   }
+  bool is_valid() const { return _valid; }
+  const char* what() const { return _bad; }
```

Comment:
I suggest to either keep the old format with the return on a new line or add a new line between `phase_name_set()` and `is_valid()` to better separate it.

Reply:
OK, reverted to old format with return on new line and new line separating the functions.